sphinx_qt_documentation/__init__.py

```diff
@@ -121,6 +121,8 @@ def missing_reference(
                         obj_type_name
                     ][target_name]
                     uri = uri.replace("##", "#")
+                    if name == 'enum':
+                        uri += "-enum"
```

Comment:
This enum add is need only when documentation point to Qt docs. If selected target is Pyside2 then link is:
https://doc.qt.io/qtforpython/PySide2/QtCore/QEvent.html#PySide2.QtCore.PySide2.QtCore.QEvent.Type
and for PyQt5 it is:
https://www.riverbankcomputing.com/static/Docs/PyQt5/api/qtcore/qevent.html#Type

so move it here:
https://github.com/Czaki/sphinx-qt-documentation/blob/76b6f6259f171dfaebe27cf9a13c8415f954c52a/sphinx_qt_documentation/__init__.py#L133-L135